docs/docsite/rst/plugins/action.rst

```diff
@@ -30,11 +30,6 @@ Plugin List
 You can use ``ansible-doc -t cache -l`` to see the list of available plugins.
 Use ``ansible-doc -t cache <plugin name>`` to see specific documentation and examples.
 
-.. toctree:: :maxdepth: 1
-    :glob:
-
-    action/*
```

Comment:
Ah, makes sense we aren't currently generating docs for `action` plugins

`DOC_PLUGINS ?= cache callback connection inventory lookup shell strategy vars`

This is because the action plugins don't currently have DOCUMENTATION blocks

Reply:
So Matt's change is correct. Above is just added in case reviews wonder *why* it's needed